hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/RemoteDMRAdd.java

```diff
@@ -40,9 +45,15 @@ private RemoteDMRAdd() {
     }
 
     @Override
+    protected void performRuntime(OperationContext context, ModelNode operation, ModelNode model,
```

Comment:
can you pull up all these performRuntime methods you had to add in the individual classes into MonitorServiceAddStepHandler? That way, it will only be a single place to add this and not have to touch the subclasses for this.